### ISSUE TITLE
Update python-dateutil dependency

### ISSUE THREAD

**Issue by brianmcmichael** [jjkester/django-auditlog#205]
Hi! We're getting the following `safety check` error. Looks like there's a new version of `python-dateutil`

```
django-auditlog 0.4.5 has requirement python-dateutil==2.6.0, but you'll have python-dateutil 2.7.5 which is incompatible.
```

**Comment by dalys** [comment]
There's already a PR for this. Unfortunately the author is required to approve it, but it seems the author has disappeared. See https://github.com/jjkester/django-auditlog/pull/201

**Comment by wearebasti** [comment]
I forked this repository for now https://github.com/alasco-tech/django-auditlog/releases and integrated the changes of @tr33house in #201 

Should be installable via a 
`pip install git+https://github.com/alasco-tech/django-auditlog.git@36eab26779bc8b8cf0b57d0007b73c7ad8ba8b67`

ATTENTION: No warranties of any kind! Also make sure to reference the commit hash as we plan to do some more changes :)

PS: Very happy to use this library again if @jjkester comes back!

**Comment by charleswhchan** [comment]
Also link to Issue #201 and PR #184.

I wish @jjkester can approve it (or allow additional maintainers for this project) as I am sure many people in the community depends on and enjoys using django-auditlog :+1:

**Comment by horida** [comment]
@jjkester any news on this? It's half a year "due"

**Comment by jjkester** [comment]
Addressed in #257 .
